modules/tests/programs/fish.nix

```diff
@@ -38,6 +41,10 @@
           machine.succeed("su bob -c 'fish -c \"abbr --query foo\" '")
           machine.fail("su bob -c 'fish -c \"abbr --query missing\" '")
 
+      with subtest("Validate aliases"):
+          machine.succeed("su bob -c 'fish -c \"alias | grep ping\" '")
+          machine.fail("su bob -c 'fish -c \"alias | grep missing\" '")
```

Comment:
I don't think this is particularly relevant here. Why would an alias `missing` be present here at all? I did not catch that on the initial fish PR, but I'm struggling to understand the purpose of this here. I get that you're trying to check for aliases not being created out of thin air, but what if we checked instead for anything that is not ping?

I think this would make more sense.

Reply:
I think the idea of this test makes a lot of sense, as we dont want random aliases to be created. A fail test on everything not ping here would work really well, as this would catch logical issues in our generation function. What do you think?